tools/log-collector/log_collector.go

```diff
@@ -229,25 +100,15 @@ func (l *LogCollector) getResourceObjects(resourcePath string, resource *apiv1.A
 	listPath := fmt.Sprintf("%s/%s", resourcePath, resource.Name)
 	err := l.disClient.RESTClient().Get().AbsPath(listPath).Do(context.TODO()).Into(&objects)
 	if err != nil {
-		if errors.IsNotFound(err) || errors.IsForbidden(err) {
+		if apierrors.IsNotFound(err) || apierrors.IsForbidden(err) {
+			log.Warnf("%s", err.Error())
 			return objects
 		}
 		return unstructured.UnstructuredList{}
 	}
 	return objects
 }
 
-// getResourceObjects returns list of objects for given resource_path
-func (l *LogCollector) getGVResourceObjects(gvResourceMap map[string]apiv1.APIResource) unstructured.UnstructuredList {
-
-	resourceObject := unstructured.UnstructuredList{}
-	for gv := range gvResourceMap {
-		gvResource := gvResourceMap[gv]
-		resourceObject.Items = append(resourceObject.Items, l.getResourceObjects(getAPIGroupVersionResourcePath(gv), &gvResource).Items...)
-	}
-	return resourceObject
-}
-
 // writeEvents writes events
 func (l *LogCollector) writeEvents(events map[string][]map[string]interface{}) error {
```

Comment:
rename to `writeEventsToFile` .. do similar for other write funcs

Reply:
resolved